autoscale-api/src/main/java/com/sequenceiq/periscope/api/endpoint/validator/ScalingConfigurationRequestValidator.java

```diff
@@ -0,0 +1,35 @@
+package com.sequenceiq.periscope.api.endpoint.validator;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+import com.sequenceiq.cloudbreak.validation.ValidatorUtil;
+import com.sequenceiq.periscope.api.model.ScalingConfigurationRequest;
+
+public class ScalingConfigurationRequestValidator implements ConstraintValidator<ValidScalingConfiguration, ScalingConfigurationRequest> {
+
+    @Override
+    public void initialize(ValidScalingConfiguration constraintAnnotation) {
+    }
+
+    @Override
+    public boolean isValid(ScalingConfigurationRequest request, ConstraintValidatorContext context) {
+        boolean validRequest = true;
+        if (request.getMaxSize() < request.getMinSize()) {
+            validRequest = false;
+            String message = String.format("The specified ScalingConfiguration is not valid because the minimum cluster size can not be more than"
+                    + " maximum cluster size.");
+            ValidatorUtil.addConstraintViolation(context, message, "status");
```

Comment:
please extract the propertyModel value (which in this case is "status") to a static final variable instead of duplicating it 3 times